reference/tax_provider.yml

```diff
@@ -25,6 +25,10 @@ paths:
       description: |-
         Submit the quote request to retrieve an estimate from the enabled third-party tax provider. Estimates are not expected to be persisted by the tax provider.
 
+        > Server URL
+        > - For supporting tax providers, the server url contains the tax provider's profile field; for example, `your_profile.example.com`.
+        > - The Send a Test Request feature is not currently supported for this endpoint.
```

Comment:
```suggestion
        > - The Try it feature is not currently supported for this endpoint.
```

Reply:
I see that we still say "Send a Test Request" on the [Storefront API pages](https://developer.bigcommerce.com/docs/rest-storefront/carts#get-a-cart), so if we plan on changing all of them to "Try It", we should do it on a separate ticket / PR (keep things consistent across all docs)

Reply:
Curious about `Try it`. We have a Test Connection feature which I think would make sense in this context.

Reply:
@benpratt77  Do you mean the Test Connection button in the UI? (the Try It is a feature on our website that allows the public to make test requests to the Tax Provider API endpoint, sending request bodies and getting response bodies back)

I'm going to merge this PR so that we can close out the GTM for Tax Profile Option, but I'm interested in adding a note about the Test Connection somewhere into the doc

Reply:
ah of course. yeah makes sense. So this text refers to the API feature. 👍